services/webinar/utils.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import fetch, { RequestInit } from 'node-fetch';
+
+export const apiBase = 'https://api.getgo.com';
```

Comment:
What's the benefit of separating out `apiBase`?  `url` in register_participant.js is very specific

Reply:
benefit - used in 2 places!